### ISSUE TITLE
Background tasks suck in Realm Database CRUD

### ISSUE THREAD

**Issue by Mohsens22** [MahStudio/Denna#53]

**Comment by Mohsens22** [comment]
Native debugging needed

**Comment by Mohsens22** [comment]
![image](https://user-images.githubusercontent.com/22152065/44612879-a1fe8800-a821-11e8-938b-bcb7110fb5a1.png)

'backgroundTaskHost.exe' (CoreCLR: CoreCLR_UWP_Domain): Loaded 'C:\Users\mohse\source\repos\Denna2\Denna\Denna\bin\x86\Debug\AppX\System.Linq.Queryable.dll'. Skipped loading symbols. Module is optimized and the debugger option 'Just My Code' is enabled.
Exception thrown: 'System.Exception' in Realm.dll
An exception of type 'System.Exception' occurred in Realm.dll but was not handled in user code
Realm accessed from incorrect thread.

The thread 0x74e8 has exited with code 0 (0x0).
The thread 0x5cc0 has exited with code 0 (0x0).
The thread 0x7664 has exited with code 0 (0x0).
The thread 0x453c has exited with code 0 (0x0).
The thread 0x14b8 has exited with code 0 (0x0).
The thread 0x788c has exited with code 0 (0x0).
The thread 0x7a2c has exited with code 0 (0x0).
The thread 0x75ec has exited with code 0 (0x0).
The thread 0x3344 has exited with code 0 (0x0).
The thread 0x79f8 has exited with code 0 (0x0).
The thread 0x1ff0 has exited with code 0 (0x0).
The thread 0x4a98 has exited with code 0 (0x0).
The program '[11700] backgroundTaskHost.exe' has exited with code 1 (0x1).

**Comment by rmclassic** [comment]
@Mohsens22 Can you code that it checks if the app is open, it passes the todo to the app and let the app add it?

**Comment by Mohsens22** [comment]
@rmclassic Yes but I need to investigate issue and fix threading. Then we can. Go for further things
